pages/settings.js

```diff
@@ -1,23 +1,48 @@
 import React from 'react'
 
-import { Icon } from '../components/common'
 import renderPage from '../renderPage'
 import {
   doApiRequest,
   formatResponse,
-  API_BASE_URL,
-  ROLE_OFFICER,
 } from '../utils'
-import { CLUBS_GREY_LIGHT } from '../constants/colors'
-import { Link } from '../routes'
-import Form from '../components/Form'
+import { DARK_GRAY } from '../constants/colors'
+import s from 'styled-components'
+import TabView from '../components/TabView'
+import ClubTab from '../components/Settings/ClubTab'
+import FavoritesTab from '../components/Settings/FavoritesTab'
+import ProfileTab from '../components/Settings/ProfileTab'
 
-class SettingsForm extends React.Component {
+const Header = s.div`
+  width: 470px;
+  height: 72px;
+  font-family: HelveticaNeue;
+  font-size: 45px;
+  font-weight: bold;
+  color: ${DARK_GRAY};
+  margin-bottom: 1rem;
+
+`
+const Notification = s.span`
+  border-radius: 4px;
+  background-color: #c5e7f7;
+  font-size: 16px;
+  font-weight: 500;
+  padding: 5px;
+  width: 500px;
+  float: right;
```

Comment:
Why use float here? Could be better to do
```
position: absolute;
right: 1rem;
```
Or something along those lines—lmk what you think works best!